packages/proxmark3/build.sh

```diff
@@ -10,11 +10,12 @@ TERMUX_PKG_BUILD_IN_SRC="true"
 TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
 
 termux_step_post_configure() {
-        LDLIBS="-L${TERMUX_PREFIX}/lib" INCLUDES="-I${TERMUX_PREFIX}/include"
-	TERMUX_PKG_EXTRA_MAKE_ARGS="client CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 SKIPGPU=1 PLATFORM=PM3GENERIC"
+	export LDLIBS="-L${TERMUX_PREFIX}/lib"
+	export INCLUDES="-I${TERMUX_PREFIX}/include"
+	TERMUX_PKG_EXTRA_MAKE_ARGS="client CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 PLATFORM=PM3GENERIC"
 }
 
 termux_step_make_install() {
-        install -Dm700 "$TERMUX_PKG_BUILDDIR"/client/proxmark3 \
-                "$TERMUX_PREFIX"/bin/proxmark3
+	install -Dm700 "$TERMUX_PKG_BUILDDIR"/client/proxmark3 \
+		"$TERMUX_PREFIX"/bin/proxmark3
```

Comment:
Why this change included with wordexp commit?

Reply:
Oh... rebase problems....